client/src/utils/data-formats/formats/bars.ts

```diff
@@ -94,13 +95,26 @@ export function bars(
         series[value] = true;
       }
     });
+    
+    // In case all keys are dates - order it and transform the datetimes
+    if (_.keys(barValues).every((value) => !Number.isNaN(Date.parse(value)))) {
```

Comment:
Could this be a "heavy" operation on big amounts of data? in which case, a better strategy would be sampling the data and disregarding excess rows, or something simpler then `Date.parse`? 
Alternatively, change result mapping from Kusto to parse as date such columns in advance